dwave_networkx/drawing/qubit_layout.py

```diff
@@ -424,9 +469,9 @@ def unoverlapped_embedding(G, emb, interaction_edges):
 
 
 def draw_yield(G, layout, perfect_graph, unused_color=(0.9,0.9,0.9,1.0),
```

Comment:
@boothby, I'm seeing something weird with this function though it might be because I'm using it directly rather than through the public functions as intended, where it might not affect anything. The following code snippet, 

```
import dwave_networkx as dnx
import matplotlib.pyplot as plt
from dwave_networkx.drawing.qubit_layout import draw_yield
G = dnx.chimera_graph(1)
layout = dnx.chimera_layout(G)
draw_yield(G.subgraph([0, 4, 5, 6, 7]), layout, G, unused_color="green", fault_color="red")
```

generates the nodes with/without faults correctly but the edges are offset from the nodes:

![image](https://user-images.githubusercontent.com/34041130/215887591-ce1c60f6-b03f-4361-b103-95a248a0ec82.png)

Reply:
I'm not reproducing that on my end (matplotlib 3.4.1, networkx 2.4).  Can you try the public interface?

Reply:
Okay, I finally managed to reproduce it through the public interface:

1. It does not happen for matplotlib 3.4.1, networkx 2.4; it happens if you upgrade to matplotlib 3.6.3 (dwave-netwokx doesn't set a version requirement for Matplotlib so users might have more recent versions)
2. We need to add ``draw_chimera_yield`` and ``draw_pegasus_yield`` to [the docs](https://docs.ocean.dwavesys.com/en/stable/docs_dnx/reference/drawing.html), currently there's only ``draw_zephyr_yield``

```
import dwave_networkx as dnx
import matplotlib.pyplot as plt
from dwave_networkx.drawing.qubit_layout import draw_embedding
G = dnx.chimera_graph(1)
layout = dnx.chimera_layout(G)
fig, ax = plt.subplots(figsize=(10, 5))
dnx.draw_chimera_yield(G.subgraph([0, 4, 5, 6, 7]), unused_color="green", fault_color="red", ax=ax)
plt.show()
```

Shows up once you zoom in as: 

![image](https://user-images.githubusercontent.com/34041130/216078902-3f1763ac-a85b-4143-8ceb-281ae5a7b889.png)